internal/server/server.go

```diff
@@ -0,0 +1,98 @@
+package server
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	_ "example.com/tfgrid-kyc-service/api/docs"
+	"example.com/tfgrid-kyc-service/internal/clients/idenfy"
+	"example.com/tfgrid-kyc-service/internal/clients/substrate"
+	"example.com/tfgrid-kyc-service/internal/configs"
+	"example.com/tfgrid-kyc-service/internal/handlers"
+	"example.com/tfgrid-kyc-service/internal/middleware"
+	"example.com/tfgrid-kyc-service/internal/repository"
+	"example.com/tfgrid-kyc-service/internal/services"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/helmet"
+	"github.com/gofiber/fiber/v2/middleware/limiter"
+	"github.com/gofiber/fiber/v2/middleware/recover"
+	"github.com/gofiber/swagger"
+)
+
+// implement server struct that have fiber app and config
+type Server struct {
+	app    *fiber.App
+	config *configs.Config
+}
+
+func New(config *configs.Config) *Server {
```

Comment:
split this function to multiple ones and call them here e.g 

registerMiddlewares
setupHandlers